src/Signer.gs

```diff
@@ -267,7 +267,7 @@
     var header_value = 'OAuth ';
 
     for(var key in oauth_data) {
-      if (key.indexOf('oauth_') === -1)
+      if (key !== 'realm' && key.indexOf('oauth_') === -1)
```

Comment:
Please add a line to the comment block on line 27, that tries to keep a log of the ways in which this file has been updated from the original.

Reply:
It looks like the original library has added realm parameter support too https://github.com/ddo/oauth-1.0a/releases/tag/2.2.0 - please let me know how to better document the change.

Reply:
Good catch. As mentioned, just add a line to the comment on line 27. I've opened issue #45 to track upgrading to the latest version of that library.